exercises/pascals-triangle/canonical-data.json

```diff
@@ -75,7 +82,18 @@
       "input": {
         "count": 10
       },
-      "expected": [[1], [1, 1], [1, 2, 1], [1, 3, 3, 1], [1, 4, 6, 4, 1], [1, 5, 10, 10, 5, 1], [1, 6, 15, 20, 15, 6, 1], [1, 7, 21, 35, 35, 21, 7, 1], [1, 8, 28, 56, 70, 56, 28, 8, 1], [1, 9, 36, 84, 126, 126, 84, 36, 9, 1]]
+      "expected": [
+        [1],
+        [1, 1],
+        [1, 2, 1],
+        [1, 3, 3, 1],
+        [1, 4, 6, 4, 1],
+        [1, 5, 10, 10, 5, 1],
+        [1, 6, 15, 20, 15, 6, 1],
+        [1, 7, 21, 35, 35, 21, 7, 1],
+        [1, 8, 28, 56, 70, 56, 28, 8, 1],
+        [1, 9, 36, 84, 126, 126, 84, 36, 9, 1]
+      ]
```

Comment:
Perhaps these should _all_ be triangles in this file.

Reply:
I will note this for consideration when writing the array formatter. After it is incorporated into CI I will try applying that to pascals-triangle and see how it looks. For the time being, I let prettier do as it will.

Reply:
Sounds good.